frontend/components/notifications/management/NotificationsManagementNetwork.vue

```diff
@@ -3,8 +3,13 @@ const { t: $t } = useTranslation()
 
 const notificationsManagementStore = useNotificationsManagementStore()
 
+const { chainIdByDefault } = useRuntimeConfig().public
+
 const networks = computed(() => notificationsManagementStore.settings.networks ?? [])
-const currentNetworkId = computed(() => notificationsManagementStore.settings.networks[0]?.chain_id ?? 0)
+const currentNetworkId = computed(
+  () => notificationsManagementStore.settings.networks
+    .find(network => network.chain_id === Number(chainIdByDefault)) ?? 0,
```

Comment:
if `NUXT_PUBLIC_CHAIN_ID_BY_DEFAULT` is not available, setting `chain id` to `0` is might not be a good idea....would `1` be a better choice?

Reply:
using chain id 0 will throw an error on the api layer, setting it to 1 is not a good idea, as it will ultimately show wrong data.